.github/workflows/release.yaml

```diff
@@ -11,54 +11,23 @@ concurrency: ${{ github.workflow }}-${{ github.ref }}
 jobs:
   release:
     name: Open Release PR or Publish Packages
-    runs-on: ['self-hosted', 'org', 'npm-publish']
     permissions:
       contents: write
-      id-token: write
+      actions: read
       pull-requests: write
+      security-events: write
+      attestations: write
+      id-token: write
       repository-projects: write
-    outputs:
-      publishedPackages: ${{ steps.changesets.outputs.publishedPackages }}
-      published: ${{ steps.changesets.outputs.published }}
```

Comment:
hmm we need these outputs. for subsequent steps like docs and homebrew releasing